app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
- [ ] Good job so far :clap:  But according to the project requirement, the comments controller should have new and create actions similar to post controller down :arrow_down:  . Please fix it to make your app work as expected :+1: